redis/src/main/java/org/apache/zeppelin/redis/RedisInterpreter.java

```diff
@@ -0,0 +1,98 @@
+package org.apache.zeppelin.redis;
+
+import org.apache.zeppelin.interpreter.Interpreter;
+import org.apache.zeppelin.interpreter.InterpreterContext;
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.ERROR;
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.SUCCESS;
+
+public class RedisInterpreter extends Interpreter {
```

Comment:
Could you add Clean header with a description of the interpreter? that will be useful for futur commiters to understand the purpose of the class